move-mutation-test/README.md

```diff
@@ -1,3 +1,151 @@
-# Move Mutation Test tool
+# Move Mutation Tester tool
 
-TODO: add docs :)
+## Summary
+
+The tool used to test the quality of the test suite and the source code.
+
+## Overview
+
+The program logic is quite simple, the tool works using the following principles:
+1. Runs tests on the original source code to ensure the original tests are valid.
+2. Internally runs the _Move Mutator_ tool to generate mutants.
+3. Runs the tests for all mutants to check if the mutants are killed by the original test suite.
+
+If the mutants are not killed, it might indicate the quality of the test suite could be improved, or in some rare cases, it might indicate an error in the original source code.
+
+**Move Mutation Tester** tool can be used on Move packages (projects) which can compile successfully and have valid tests that are passing.
+Using filters it is possible to run the tool only on certain mutants filtered by:
+ - Module name (`--mutate-modules` argument)
+ - Function name (`--mutate-functions` argument)
+
+The tool cannot be used with single Move files.
```

Comment:
Maybe just shortly, why it cannot? (I know 😸 )

Reply:
Done